config/locales/diaspora/en.yml

```diff
@@ -346,6 +346,10 @@ en:
     tutorials: "tutorials"
     tutorial: "tutorial"
     irc: "IRC"
+    closed_pod: "This pod is currently closed to new registrations. However, you can still join the diaspora* network by registering on %{wiki}. Because all pods are interconnected, you will have access to the same content there."
+    find_pods: "There’s a list of pods you can sign up to at %{poduptime}."
+    other_questions: "If you have any other questions regarding choosing a pod, checkout our %{wiki}."
```

Comment:
Total nit-pick, and apologies I didn't notice this before, but 'checkout' should be 'check out' (with a space).

Reply:
Fixed.